### PR TITLE
[ko] Korean Translation for 2-Farm lesson1

### PR DESCRIPTION
**Pull request by jhYun505**
Add
- README.ko.md
- pi-temp.ko.md
- assignment.ko.md
- single-board-computer-temp-publish.ko.md

### CODE REVIEW

File: 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+   이 프로젝트의 폴더명을 입력하십시오.
+   `temperature-sensor-server`.
+
+1. `client_name`이 이 프로젝트를 반영하는 지 확인하십시오.
+
+   ```cpp
+   client_name = id + 'temperature_sensor_server'
+   ```
+
+1. 아래의 import 목록을 보고 파일의 상단에 import 하는 부분을 추가하십시오:
```
Comment (DobiIsFree) [inline 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md:174]: 번호를 고쳐주세요

File: 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+디지털 농업은 농업 데이터를 수집, 저장, 분석하는 도구를 사용하여 우리가 농사를 짓는 방식을 변화시키고 있습니다. 우리는 현재 세계 경제 포럼(WEF)이 '4차 산업 혁명'이라 설명하는 시기에 살고 있고, 디지털 농업의 증가는 '4차 농업 혁명' 또는 '농업 4.0'이라고 불리게 되었습니다.
+
+> 🎓 디지털 농업이라는 용어는 또한 농장에서 식탁까지 가는 전체 여정인 '농업 가치사슬'도 포함합니다. 이는 식품 배송 및 가공, 창고 및 전자 상거래 시스템, 심지어 트랙터 대여 앱에 대한 제품 품질 추적을 포함합니다!
+
+이러한 변화를 통해 농부들은 생산량을 증가시키고, 더 적은 비료와 농약을 사용하고, 더 효율적으로 급수할 수 있었습니다. 주로 부유한 나라에서 사용하지만, 센서와 다른 장치들은 천천히 가격을 내리고 있고 개발도상국에서도 더 접근이 용이하도록 만들고 있습니다.
+
+디지털 농업에서 사용 가능한 몇 가지 기술:
+
+- 온도 측정 - 온도를 측정하는 것은 농부들에게 식물의 성장과 성숙도를 예측할 수 있게 합니다.
+- 자동 관수 - 토양의 수분을 측정하고 토양이 너무 건조할 때, 시간을 정헤 물을 주는 대신 관개 시스템을 켭니다. 시간을 정해 물을 주면 농작물이 더운 기간이나 가뭄 동안 물이 부족하거나, 비가 오는 동안 물이 너무 많아질 수 있습니다. 토양이 필요로 할 때만 물을 줌으로써 농부들은 물 사용을 최적화할 수 있습니다.
```
Comment (DobiIsFree) [inline 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md:34]: '정헤' 오타 발견!!

File: 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+
+식물은 일 평균 기온을 기준으로 기준 혹은 최저 온도와 적정 온도, 최고 온도를 가지고 있습니다.
+
+- 기준 온도 - 식물이 자라는 데에 필요한 최소한의 일 평균 온도입니다.
+- 적정 온도 - 식물이 가장 많이 자라게 하는 최적의 일 평균 온도입니다.
+- 최고 온도 - 식물이 견딜 수 있는 최고 일 평균 온도입니다. 이 온도 이상에서 식물은 물을 아끼고 살아 남기 위해 스스로의 성장을 멈춥니다.
+
+> 💁 이것들은 일간 및 야간 온도에 대한 평균을 낸 평균 온도입니다. 식물은 또한 더 효율적으로 광합성을 하고 밤에 에너지를 절약하기 위해 밤낮으로 다른 온도가 필요합니다.
+
+Each species of plant has different values for their base, optimal and maximum. This is why some plants thrive in hot countries, and others in colder countries.
```
Comment (WZNT-KimHyeJin) [inline 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md:55]: Please remove this line :)

File: 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+
+식물은 정해진 날짜에 자라지 않습니다. 예를 들어 씨앗을 심고 정확히 100일 후에 열매를 맺는다는 것을 알 수는 없습니다. 대신 농부는 식물이 자라는 데 걸리는 시간을 대략적으로 알 수 있으며 매일 작물이 준비되는 시간을 확인합니다.
+
+이는 대규모 농장의 노동력에 막대한 영향을 끼칩니다. 그리고 농부들이 예상치 못하게 일찍 준비된 작물을 놓치는 위험을 줍니다. 농부는 기온을 측정하여 식물이 받은 GDD를 계산할 수 있고, 그들이 예상 성숙도와 가까워졌는지 확인 하도록 합니다.
+
+IoT 장치를 이용하여 온도 데이터를 수집하면, 농부는 자동적으로 식물이 성숙에 가까워졌는지 알 수 있습니다. 이러한 일일반적인 아키텍처는 IoT 장치가 온도를 측정한 다음 MQTT와 같은 것을 사용하여 인터넷을 통해 이 원격 측정 데이터를 게시하도록 하는 것입니다. 서버 코드는 이 데이터를 수신하고 이것을 데이터베이스와 같은 어딘가에 저장합니다. 이것은 데이터가 나중에 분석될 수 있다는 것을 의미합니다. 예를 들어, 야간 작업을 통해 하루의 GDD를 계산하고, 지금까지 각 작물에 대한 GDD를 합산하고, 식물이 성숙에 가까워지면 알림을 보낼 수 있습니다.
+
+![원격 측정 데이터는 서버로 보내져 데이터베이스에 저장됩니다.](../../../../images/save-telemetry-database.png)
+
+서버 코드는 또한 부가 정보를 추가함으로써 데이터를 보강할 수 있습니다. 예를 들어, IoT 장치는 어떤 장치인지 나타내기 위해 식별자를 게시할 수 있으며, 서버 코드는 이를 사용ㅎ여 장치의 위치와 모니터링 중인 작물을 조회할 수 있습니다.또한 일부 IoT 장치에는 정확한 시간을 추적하는 데 필요한 하드웨어가 없거나 인터넷을 통해 현재 시간을 읽는 데 추가 코드가 필요하므로 현재 시간과 같은 기본 데이터를 추가할 수 있습니다.
```
Comment (WZNT-KimHyeJin) [inline 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md:136]: You can check the 사용 ㅎ여

File: 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+
+이를 측정할 수 있는 센서가 있는지 찾아보십시오. 이러한 레벨을 제어하는 액추에이터는 어떻습니까? 식물의 성장을 최적화하기 위해 하나 이상의 IoT 장치를 어떻게 조합해야 할까요?
+
+## 강의 후 퀴즈
+
+[강의 후 퀴즈](https://black-meadow-040d15503.1.azurestaticapps.net/quiz/10)
+
+## 복습 및 독학
+
+- 디지털 농업에 대해 [디지털 농업에 대한 Wikipedia 페이지](https://wikipedia.org/wiki/Digital_agriculture)에서 더 읽어보십시오. 정밀한 농업에 대해서는 [정밀 농업에 대한 Wikipedia 페이지](https://wikipedia.org/wiki/Precision_agriculture)에서 읽어보십시오.
```
Comment (WZNT-KimHyeJin) [inline 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md:263]: I think you have to choose 정밀한 농업 or 정밀 농업

**Review by WZNT-KimHyeJin — COMMENTED** [review]
You did great job!!

File: 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md (hunk 2 above)
Comment (jhYun505) [reply 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md]: Thank you

File: 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md (hunk 3 above)
Comment (jhYun505) [reply 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md]: I forgot it! Thanks

File: 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md (hunk 5 above)
Comment (jhYun505) [reply 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md]: I picked 정밀 농업
thank you

Comment (WZNT-KimHyeJin) [reply 2-farm/lessons/1-predict-plant-growth/translations/README.ko.md]: I think you have to click the resolve button, hasn't it?

**Comment by jimbobbennett** [comment]
Is this ready to merge?

**Comment by jhYun505** [comment]
> Is this ready to merge?

yes it is!

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**